### ISSUE TITLE
Add OpenWeather support

### ISSUE THREAD

**Issue by BlitDev** [NayamAmarshe/please#15]
It takes a long time to visit various sites, I spend much more time in the terminal. And if the weather greets me at every launch, it will be just fine :)
After installing **please**, you will need to insert your **Weather token** and configure **your location**. By default, ***the city should be set, according to the time zone***, if this is not possible, any city is set.

**Comment by NayamAmarshe** [comment]
Yes, that's actually a thing we were planning to add. The hardest part would be handling synchronous queries, which would make the startup slower so have to think of a way around that first before we start with the implementation.

**Comment by NayamAmarshe** [comment]
This was planned but I'm currently putting this on hold. The main issue is asynchronous queries cannot be performed with terminals. Maybe I'll come back to this but any PRs are appreciated :)
